source/develop/release-management/features/migration-enhancements.html.md

```diff
@@ -1,11 +1,14 @@
 ---
 title: Migration Enhancements
 category: feature
-authors: mpolednik, mskrivan, ofrenkel, sandrobonazzola, tjelinek
+authors: tjelinek,mbetak,jniederm,vfeenstr
```

Comment:
We can't change anything in the wiki header, this is legacy content that shouldn't be touched and we need it for the archives. Please revert and update the PR accordingly.